Arrowgene.Ddon.GameServer/Characters/ItemManager.cs

```diff
@@ -210,11 +211,11 @@ private CDataItemUpdateResult ConsumeItem(DdonServer<GameClient> server, Charact
             ntcData.ItemList.Color = item.Color;
             ntcData.ItemList.PlusValue = item.PlusValue;
             ntcData.ItemList.Bind = false;
-            ntcData.ItemList.EquipPoint = 0;
+            ntcData.ItemList.EquipPoint = item.EquipPoints;
             ntcData.ItemList.EquipCharacterID = 0;
             ntcData.ItemList.EquipPawnID = 0;
             ntcData.ItemList.WeaponCrestDataList = item.WeaponCrestDataList;
-            ntcData.ItemList.ArmorCrestDataList = item.ArmorCrestDataList;
+            ntcData.ItemList.AddStatusData = item.AddStatusParamList;
             ntcData.ItemList.EquipElementParamList = item.EquipElementParamList;
             ntcData.UpdateItemNum = -finalConsumeNum;
```

Comment:
I am not sure if `UpdateItemNum` should be negative here. None of the pcap data shows this as a negative number.